Tests/UnitTests/Networking/ETagManagerTests.swift

```diff
@@ -86,6 +86,30 @@ class ETagManagerTests: TestCase {
         expect(response?.requestDate) == requestDate
     }
 
+    func testStoredResponseIsNotUsedIfResponseCodeIs304ButContainsNoETag() throws {
+        let eTag = "an_etag"
+        let url = self.urlForTest()
+        let request = URLRequest(url: url)
+        _ = self.mockStoredETagResponse(for: url, statusCode: .success, eTag: eTag)
+        let responseObject = try JSONSerialization.data(withJSONObject: ["a": "response"])
+        let requestDate = Date().addingTimeInterval(-100000)
```

Comment:
why's the old request date needed?

Reply:
Just to test that the request date returned matches the one from the response.